docs/3.0.0/docs/tools/daml-shell/index.rst

```diff
@@ -0,0 +1,454 @@
+         * exercise - choice by event-id
+         * exercises - exercised choices
+         * go - upper offset bound
+         * net-changes - net contract counts
+         * set - configure application settings
+         * transaction - transaction details
+
+FAQ
+===
+
+   **Why do I not see any archived contracts?**
+
+This is likely because PQS has been configured to seed the database from
+the ACS, which does not include historical offsets, archived contracts,
+or exercised choices.
+
+To see pre-existing archived contracts, the database needs to be seeded
+from the ``Transaction Stream`` or ``Transaction Tree Stream``.
+
+Solution: Set ``--pipeline-ledger-start`` to ``Genesis`` when running
+PQS for the first time (refer to the `PQS
+documentation <https://docs.daml.com/query/pqs-user-guide.html>`__ or
+``--help`` output).
+
+   **Why do I not see any choices?**
+
+Choices are only visible on the Ledger API’s
+``Transaction Tree Stream``. Set ``--pipeline-datasource`` to
+``TransactionTreeStream`` when running PQS (refer to the `PQS
+documentation <https://docs.daml.com/query/pqs-user-guide.html>`__ or
+``--help`` output).
+
+   **Why do I still not see any choices?**
+
+See ``Why do I not see any archived contracts`` above.
+
+   **Why do I not see any interface views?**
+
+Interfaces are only visible on the Ledger API’s ``Transaction Stream``
+or ACS (not the ``Transaction Tree Stream``).
+
+Set ``--pipeline-datasource`` to ``TransactionStream`` when running PQS
+(refer to the `PQS
+documentation <https://docs.daml.com/query/pqs-user-guide.html>`__ or
+``--help`` output).
+
+   **Why do all contracts show the same ledger offset?**
+
+See ``Why do I not see any archived contracts`` above.# Configuration
+
+Daml Shell can be configured via CLI arguments, environment variables,
+through a configuration file, or interactively using the ``set``
+command.
+
+Command line arguments
+----------------------
+
+To see available command line options, run Daml Shell with the
+``--help`` flag:
+
+.. code:: sh
+
+   Usage: daml-shell [options]
+
+     -h, --help               prints this usage text
+     --config <file>          HOCON config file with settings as key-value pairs
+     --wildcard-char <char>   wildcard character used to abbreviate identifiers. Default: "…"
+     --identifier-hash-length <length>
+                              Number of characters to render for hashes inside identifiers. Default: 20
+     --full-identifiers       Disable identifier shortening.
+     --identifier-trim-location <leading|middle|trailing>
+                              Where to trim long identifiers. Default: "trailing"
+     --disable-color          Disable ANSI colored output.
+
+Configuration file
+------------------
+
+These configuration parameters can also be set in a reusable
+configuration file in HOCON (a JSON superset) using the flag
+``--config <filename>``.
+
+Example file content:
+
+.. code:: hocon
```

Comment:
```suggestion
Configuration file
==================

You can set Daml Shell parameters in a reusable
configuration file in HOCON (a JSON superset) using the flag
``--config <filename>``.

Example file content:

.. code-block:: text
```

Reply:
I chose `text` for now, but you can use any of the Pygments lexers: https://pygments.org/docs/lexers/